easybuild/easyconfigs/p/PGI/PGI-20.1-GCC-9.2.0-2.32.eb

```diff
@@ -0,0 +1,28 @@
+# Contribution from the Crick HPC team
+# uploaded by J. Sassmannshausen
+
+name = 'PGI'
+version = '20.1'
+
+homepage = 'https://www.pgroup.com/'
+description = "C, C++ and Fortran compilers from The Portland Group - PGI"
+
+toolchain = SYSTEM
+
+sources = ['pgilinux-20%(version_major)s-%(version_major)s%(version_minor)s-x86-64.tar.gz']
+checksums = ['996c76867062a41361fcac16784c3b39fdb744e6a8c18d436ec224f062ada4d8']
+
+local_gccver = '9.2.0'
+local_binutilsver = '2.32'
+versionsuffix = '-GCC-%s-%s' % (local_gccver, local_binutilsver)
```

Comment:
Do we want to keep it this way, or use GCC-9.2.0 without specifying binutils version like we do for most everything else now??

Reply:
Yeah, I should change that part...

Reply:
And do we even want to drop even the GCC suffix? we did for iccifort... and only have a suffix for exceptions

Reply:
PGI isn't part (yet?) of the commonly used toolchains yet though, while `iccifort` is...

Reply:
No, but that has been a target of mine for a while. I just never managed to find enough time...

Reply:
But dropping the binutils suffix is good enough to start with.
We need 20.1 installed next week so deciding on the suffix today or tomorrow would be good.